cibuild.cmd

```diff
@@ -58,14 +59,19 @@ if /i "%HOST%"=="CoreCLR" (
     goto :error
 )
 
+set LOCALIZED_BUILD_ARGUMENT=
+if "%LOCALIZED_BUILD%"=="true" (
+    set LOCALIZED_BUILD_ARGUMENT="/p:LocalizedBuild=true"
+)
+
 :: Restore build tools
 call %~dp0init-tools.cmd
 
 echo.
 echo ** Rebuilding MSBuild with downloaded binaries
 
 set MSBUILDLOGPATH=%~dp0msbuild_bootstrap_build.log
-call "%~dp0build.cmd" /t:Rebuild /p:Configuration=%BUILD_CONFIGURATION% /p:"SkipBuildPackages=true"
+call "%~dp0build.cmd" /t:Rebuild /p:Configuration=%BUILD_CONFIGURATION% /p:"SkipBuildPackages=true" %LOCALIZED_BUILD_ARGUMENT%
```

Comment:
I'd keep it on. The purpose of the badly named flag is for dev experience. You don't want to do 2 builds every time. But I'd rather force localized builds every time so it's tested a lot :)